cmd/version.go

```diff
@@ -49,6 +50,17 @@ func newVersionCmd(homedir, getIstioVersion string) *cobra.Command {
 			}
 
 			logger.Infof("getistio version: %s\nactive istioctl: %s\n", getIstioVersion, cur.ToString())
+			k8sCLient, err := util.GetK8sClient()
+			if err != nil {
+				logger.Infof("no active Kubernetes clusters found\n")
+				return err
+			}
+			v, err := k8sCLient.ServerVersion()
+			if err != nil {
+				logger.Infof("cannot retrieve Kubernetes cluster server information\n")
+				return err
```

Comment:
in this case, shall we can skip `if remote {}` block rather than returning here?